src/Compilers/CSharp/Portable/Compilation/SyntaxTreeSemanticModel.cs

```diff
@@ -178,7 +178,9 @@ internal override IOperation GetOperationWorker(CSharpSyntaxNode node, Cancellat
                 case AccessorDeclarationSyntax accessor:
                     model = (accessor.Body != null || accessor.ExpressionBody != null) ? GetOrAddModel(node) : null;
                     break;
-
+                case RecordDeclarationSyntax { ParameterList: { }, PrimaryConstructorBaseType: { } } recordDeclaration when TryGetSynthesizedRecordConstructor(recordDeclaration) is SynthesizedRecordConstructor ctor:
```

Comment:
>ctor [](start = 210, length = 4)

Consider omitting the variable name so it's clear we're not expecting to use the value.

Reply:
> Consider omitting the variable name so it's clear we're not expecting to use the value.

Sometimes it is useful during debugging

---
In reply to: [461794242](https://github.com/dotnet/roslyn/pull/46308#discussion_r461794242) [](ancestors = 461794242)